wiki/Music_theory/Syncopation/de.md

```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+---
+
+# Synkope
+
+Eine **Synkope** ist die regelmäßige Akzentuierung oder Betonung von Noten an ungewöhnlichen Stellen. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks gibt einen gleichmäßigen [Rhythmus](/wiki/Music_theory/Rhythm) aus starken und schwachen Beats vor, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen versetzt werden. Beim [Beatmapping](/wiki/Beatmapping) sind Synkopen normalerweise zu erkennen, wenn [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, wie zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste.
+
+Das [Insane-Level von *Chasers - Lost* erstellt von fanzhen0019](https://osu.ppy.sh/beatmapsets/151114#osu/372628) enthält als einfaches Beispiel [Slider](/wiki/Gameplay/Hit_object/Slider)-Patterns mit [Sliderköpfen](/wiki/Gameplay/Hit_object/Slider/Sliderhead), die sowohl im Onbeat als auch im Offbeat liegen.
```

Comment:
I think we should avoid `Onbeat`:

```suggestion
Das [Insane-Level von *Chasers - Lost* erstellt von fanzhen0019](https://osu.ppy.sh/beatmapsets/151114#osu/372628) enthält als einfaches Beispiel [Slider](/wiki/Gameplay/Hit_object/Slider)-Patterns mit [Sliderköpfen](/wiki/Gameplay/Hit_object/Slider/Sliderhead), die sowohl auf dem Beat als auch auf dem Offbeat liegen.
```

Reply:
My intention was to avoid `Onbeat` as well.